app/jobs/check_layer_job.rb

```diff
@@ -4,6 +4,15 @@ class CheckLayerJob < ApplicationJob
   ##
   # @param [GeoMonitor::Layer] layer
   def perform(layer)
-    layer.check
+    # See if in Solr first, if not, deactivate and exit
+    num_found = Blacklight.default_index.connection.get(
+      'select',
+      params: { q: "layer_slug_s:#{layer.slug}", fl: '*' }
+    )['response']['numFound'].to_i
+    if num_found > 0
```

Comment:
@drh-stanford I would like your advice on this approach. This seemed like a better approach than trying to query the RDBMS with values in Solr (a very large SQL select that I think was bogging down the previous implementation).

Reply:
The Solr index is the authoritative place to look, so I think this is fine to query by the slug during the job. Although you may not need the `fl: '*'` -- how about returning only the slug itself, unless you need additional information.